eclair-core/src/main/scala/fr/acinq/eclair/io/PendingChannelsRateLimiter.scala

```diff
@@ -102,6 +101,7 @@ private class PendingChannelsRateLimiter(nodeParams: NodeParams, router: ActorRe
         context.log.info("restored {} private peers with pending channel opens.", pendingPrivateNodeChannels.size)
         pendingPrivateNodeChannels.foreach(p => context.log.debug(" {} -> {}", p._1, p._2))
         context.log.debug("stashed commands: {}", stash.size)
+        stash.foreach(p => context.log.debug(" {}", p.toString))
```

Comment:
I'm not a big fan of those debug logs, because we will iterate on large lists even when the log level isn't `debug`. I don't think they're useful, so I removed them in https://github.com/ACINQ/eclair/commit/ba59f2f943e141f99d4eb8ea0aee2521b1590949

Reply:
Agree. I can always add this logging back if we need them, but not necessary generally.